cargo-dist/src/tasks.rs

```diff
@@ -1808,8 +1810,8 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             if !other_artifacts.is_empty() && download_url.is_some() {
                 let download_url = download_url.as_ref().unwrap();
                 writeln!(gh_body, "## Download {heading_suffix}\n",).unwrap();
-                gh_body.push_str("|        |        |\n");
-                gh_body.push_str("|--------|--------|\n");
+                gh_body.push_str("|  File  | Platform | Checksum |\n");
+                gh_body.push_str("|--------|----------|----------|\n");
```

Comment:
huh interesting, I apparently didn't make this code tolerant of the fact that checksums can be disabled. Whelp, pre-existing issue.